src/app/app-routing.module.ts

```diff
@@ -28,10 +31,13 @@ const routes: Routes = [
   { path: 'about', component: AboutComponent },
   { path: '404', component: NotFoundComponent},
   { path: '', redirectTo: 'about', pathMatch: 'full' },
+  { path: 'levels', component: LevelListComponent, canActivate: [LoggedInGuard],},
+  { path: 'level/:id', component: LevelDetailComponent, canActivate: [LoggedInGuard] },
+  { path: 'topics', component: TopicListComponent, canActivate: [LoggedInGuard],}
```

Comment:
There is no link at the Navbar to acces this route.

Reply:
I think topics route is never used and neither the TopicComponent, you should remove it if it's not used.